docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -8,6 +8,7 @@
 - 2020 April 30: Switch to `Any`
 - 2020 May 14: Describe public key encoding
 - 2020 June 08: Store `TxBody` and `AuthInfo` as bytes in `SignDoc`; Document `TxRaw` as broadcast and storage type.
+- 2020 August 07: Use Regencode from ADR 026 for serializing `SignDoc`.
```

Comment:
remove Regencode here in favor of just ADR026?

Reply:
Yes, thank you. I missed this one and fixed during your review already